docs/discussions/deployment.md

```diff
@@ -6,6 +6,8 @@ In production you'll probably also want to use a process manager to be able to s
 
 You'll also want to turn off [debug mode] to disable auto-reload and prevent displaying error tracebacks in the browser.
 
+For deploying on Heroku please refer to [How To Deploy On Heroku](/docs/how-to/deploy-to-heroku) guide.
```

Comment:
Have you tested this link? I believe the `/docs` part is superfluous. Also you need to add the `.md` extension - to be confirmed. :-)

```suggestion
For deploying on Heroku please refer to [How To Deploy On Heroku](/how-to/deploy-to-heroku.md) guide.
```